ENH: Add Linux aarch64 builds

matthewfeickert opened the pull request:
* Add linux-aarch64 builds to the recipe.
* Bump build number.

Related: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7215

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [N/A] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

matthewfeickert committed:
ENH: Add Linux aarch64 builds.

* Add linux-aarch64 builds to the recipe.

matthewfeickert commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

matthewfeickert commented:
Looks like this needs rebuilds of `yoda` `v1.9.6` https://github.com/conda-forge/yoda-feedstock.

matthewfeickert committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2026.01.18.13.43.35

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.1
- rattler-build-conda-compat 1.4.10

matthewfeickert commented:
Give up on `linux-ppc64le` builds as these will be removed in the future probably.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: failed


Thus the PR was not passing and not merged.

matthewfeickert commented:
Cross-compilation fails for `cython` with `cannot check for file existence when cross compiling`

```
 │ │ checking for Cython >= 0.24.0... yes
 │ │ checking for cython-3.11... no
 │ │ checking for cython3.11... no
 │ │ checking for cython-3... no
 │ │ checking for cython3... no
 │ │ checking for cython... $BUILD_PREFIX/bin/cython
 │ │ configure: Cython >= 0.24 found at $BUILD_PREFIX/bin/cython: Python extension source can be rebuilt (for developers)
 │ │ configure: error: cannot check for file existence when cross compiling
 │ │ checking for pyext/rivet/core.cpp... 
 │ │ × error Script failed with status 1
```

matthewfeickert committed:
DEBUG: ac_cv_file_pyext_rivet_core_cpp=no

matthewfeickert committed:
not where build info is coming from

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

Commits were made to this PR after the `automerge` label was added. For security reasons, I have disabled automerge by removing the `automerge` label. Please add the `automerge` label again (or ask a maintainer to do so) if you'd like to enable automerge again!

matthewfeickert committed:
Try adding patch for cross-compilation

matthewfeickert committed:
Revert "Try adding patch for cross-compilation"

This reverts commit 9e76375eb81a3f88a454eb85f6ea2bca1d8e72c7.

matthewfeickert committed:
Try adding HepMC3 to build requirements for header file
